### DISCUSSION TITLE
{\an1} tags appearing randomly in scc

### DISCUSSION THREAD

**Discussion by kvarks** [SubtitleEdit/subtitleedit#5234] [Q&A]
Has anyone else had the problem with converting from srt file to scc: {\an1} tags just appear randomly in some subtitles. If taken away manually they reappear again after reopening the file. How to get rid of this random aligning format? Thanks!

**Comment by mongeroid** [comment]
Would also appreciate an answer for this. I randomly find them when I reopen an SCC I saved previously. Some players override alignment tags, others don't. I'm not sure if these randomly appearing {/an1} are affecting my files, but when I put them in Premiere it appears that it does.

**Comment by 7heS4ge** [comment]
{\an1} change the position of the text, it appears at the left only
{\an8} display the text on middle top (above a line compare to normal)
it's useful when the text can't appear correctly in the movie : maybe a text already exists at this position, or the background has the same color of the text.
